integration/utmp_integration_test.go

```diff
@@ -143,6 +143,9 @@ func TestRootUsernameLimit(t *testing.T) {
 	username = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
 	err = uacc.Open(utmpPath, wtmpPath, username, "localhost", host, tty)
 	require.NoError(t, err)
+
+	err = uacc.Close(utmpPath, wtmpPath, tty)
```

Comment:
I see 2 opens above but only one close, is that OK?

Should we close in a defer or t.Cleanup to make sure it always happens?

Reply:
I think it's ok.
1. The first Open is expected to fail, so we should not call close.
2. The second Open() should be closed, but only if there is no failure.

`defer` would need to be in the same place as `Close()` is right now (at the very end of the function). For me sounds like noise.

Reply:
👍